### ISSUE TITLE
Please replace with a clear and descriptive title

### ISSUE THREAD

**Issue by wkuehler** [salesforce-ux/slds-linter#302]
**Describe the bug**

slds-linter is reporting errors for known hooks.  For example: "Can't validate with unknown variable '--slds-g-color-error-container-1'.","error","css/no-invalid-properties",

<img width="1919" height="1036" alt="Image" src="https://github.com/user-attachments/assets/24b1bb0b-fcb7-4958-a2ac-8e7b3a0afaa8" />

**Configuration**
Version 1.0.4

Please enter the following command in a terminal and copy/paste its output:
```
 npx @salesforce-ux/slds-linter 
Usage: npx @salesforce-ux/slds-linter@latest [options] [command]

SLDS Linter CLI tool for linting styles and components

Options:
  -V, --version  output the version number
  -h, --help     display help for command

Commands:
  lint           Run both style and component linting
  report         Generate report from linting results
  emit           Emits the configuration files used by slds-linter cli
  help           display help for command
```

**Comment by riteshiitbbs007** [comment]
@wkuehler The issue has been fixed and will be available in the next patch release. Apologies for the inconvenience caused.
